packages/css/src/paragraph/paragraph.scss

```diff
@@ -5,28 +5,45 @@
 
 @import "../../utils/calculate-fluid-style";
 
+@import "../../node_modules/@amsterdam/design-system-tokens/dist/variables";
```

Comment:
Can we prevent mentioning `node_modules` here?

Reply:
Apparently, sass can't import from `node_modules` natively. We could use `sass-loader` I think, but that might be overkill for a simple import...